src/core/components/hv-navigator/index.tsx

```
@@ -425,17 +448,20 @@ export default class HvNavigator extends PureComponent<Props> {
   };
 
   render() {
+    const Navigator =
+      this.props.params && this.props.params.isModal
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)